metaflow/plugins/aip/aip_metaflow_step.py

```diff
@@ -352,7 +371,7 @@ def aip_metaflow_step(
         flow_name,
         is_interruptible,
     )
-    cmd: str = cmd_template.format(
+    step_cmd: str = cmd_template.format(
```

Comment:
nit: I wouldn't change the name `cmd` if it's not necessary as part of this MR...

Reply:
Cool I'll open another PR for this.